tpot/tpot.py

```diff
@@ -146,11 +147,7 @@ def __init__(self, population_size=100, generations=100,
         self._toolbox.register('mate', gp.cxOnePoint)
         self._toolbox.register('expr_mut', gp.genFull, min_=0, max_=3)
         self._toolbox.register('mutate', self._random_mutation_operator)
-
-        if not scoring_function:
-            self.scoring_function = self._balanced_accuracy
-        else:
-            self.scoring_function = scoring_function
+        self.scoring_function = scoring_function or self._balanced_accuracy
```

Comment:
I'm unsure about changes like this because it makes the code less explicit and harder to read. In fact, I'm not even sure if this line works? I'm assuming it does if you submitted the change.